src_c/_pygame.h

```diff
@@ -319,7 +315,6 @@ struct pgColorObject {
 #define PYGAMEAPI_PIXELARRAY_NUMSLOTS 2
 #define PYGAMEAPI_COLOR_NUMSLOTS 5
 #define PYGAMEAPI_MATH_NUMSLOTS 2
-#define PYGAMEAPI_CDROM_NUMSLOTS 2
```

Comment:
Does this need to reflect the number of slots used somewhere else?

Reply:
You'd think so, but it must've been removed already. I used `grep` to check the codebase for extraneous CDROM mentions.